sky/engine/web/Sky.cpp

```diff
@@ -55,8 +56,14 @@ namespace blink {
 
 namespace {
 
+base::LazyInstance<AfterMicrotasksCallback> s_afterMicrotasksCallback =
+    LAZY_INSTANCE_INITIALIZER;
+
 void didProcessTask() {
   tonic::DartMicrotaskQueue::RunMicrotasks();
+  const AfterMicrotasksCallback& callback = s_afterMicrotasksCallback.Get();
+  if (callback)
+    callback();
```

Comment:
Putting code here is fairly expensive.  I wonder if you can hook into the task observation machinery in message loop directly so that this code doesn't have any runtime cost in production.